app/Http/Requests/ImageUploadRequest.php

```diff
@@ -26,8 +27,8 @@ public function authorize()
     public function rules()
     {
         return [
-            'image' => 'mimes:png,gif,jpg,jpeg,svg',
-            'avatar' => 'mimes:png,gif,jpg,jpeg,svg',
+            'image' => 'mimes:png,gif,jpg,jpeg,svg,bmp',
+            'avatar' => 'mimes:png,gif,jpg,jpeg,svg,bmp,svg+xml',
```

Comment:
Just added this to accommodate SVGs because you know someone is gonna ask for it